inflammation/models.py

```diff
@@ -44,3 +44,13 @@ def daily_min(data):
     """
     return np.min(data, axis=0)
 
+
+def s_dev(data):
```

Comment:
s_dev is an uncommon abbreviation. a better name is standard_deviation, or std_dev.